hellomama_registration/settings.py

```diff
@@ -14,7 +14,7 @@
 import djcelery
 import dj_database_url
 import mimetypes
-
+from hellomama_registration.utils import MSG_TYPES
```

Comment:
Would it not be better to define the MSG_TYPES in settings and import them where needed rather than define them in utils? 

This might cause a circular import?